hail/src/main/scala/is/hail/annotations/UnsafeRow.scala

```diff
@@ -244,13 +244,16 @@ object SafeRow {
     a match {
       case _: UnsafeRow => false
       case _: UnsafeIndexedSeq => false
+      case _: UnsafeNDArray => false
 
       case r: Row =>
         r.toSeq.forall(isSafe)
       case a: IndexedSeq[_] =>
         a.forall(isSafe)
       case i: Interval =>
         isSafe(i.start) && isSafe(i.end)
+      case nd: NDArray =>
+        isSafe(nd.getRowMajorElements().forall(isSafe))
```

Comment:
seems wrong -- should check type of top level and then recur to elements?

Reply:
Top level is checked above right?

Reply:
oh! I see.

Should be         

`nd.getRowMajorElements().forall(isSafe))`

though, right? (still probably works as written, but does an extra isSafe on the Boolean returned by forall)